test/new-unit-tests/dune

```diff
@@ -0,0 +1,14 @@
+(library
```

Comment:
Remark on the directory name: these won't stay "new" for long, so I suggest a more descriptive name like `test/expect-tests`. what do you think?

Reply:
Seems good!